rootfs/api/models.py

```diff
@@ -790,9 +747,25 @@ class Meta:
         ordering = ['-created']
         unique_together = (('app', 'uuid'),)
 
+    @property
+    def _scheduler(self):
```

Comment:
Ah no, that's legacy, my first iteration had `Config` do updating of the RC but that's not what I ended up wanting. Removing